aspnetcore/migration/22-to-30.md

```diff
@@ -688,7 +688,7 @@ public void Configure(IApplicationBuilder app)
     });
 }
 
-public class HomeController : ControllerBase
+public class HomeController : Controller
```

Comment:
Why change this?

Reply:
> Why change `ControllerBase` to `Controller`

That's what the templates generate so I assume that's what most customers use.